<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>32</green>
        <blue>41</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>39</red>
        <green>48</green>
        <blue>61</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>40</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>16</green>
        <blue>20</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>17</red>
        <green>21</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>32</green>
        <blue>41</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>16</green>
        <blue>20</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>32</green>
        <blue>41</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>39</red>
        <green>48</green>
        <blue>61</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>40</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>16</green>
        <blue>20</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>17</red>
        <green>21</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>32</green>
        <blue>41</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>16</green>
        <blue>20</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>16</green>
        <blue>20</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>32</green>
        <blue>41</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>39</red>
        <green>48</green>
        <blue>61</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>32</red>
        <green>40</green>
        <blue>51</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>16</green>
        <blue>20</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>17</red>
        <green>21</green>
        <blue>27</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>16</green>
        <blue>20</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>13</red>
        <green>16</green>
        <blue>20</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>32</green>
        <blue>41</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>32</green>
        <blue>41</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>26</red>
        <green>32</green>
        <blue>41</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="SolidPattern">
       <color alpha="128">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget {
	background-color : rgb(26, 32, 41);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget {
	padding : 50px;
}</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <layout class="QVBoxLayout" name="verticalLayout" stretch="5,0,0,0,0,0,0">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>45</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>7</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>150</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton {
	background-color :rgb(26, 32, 41);
	border: none;
	color : white;
}
#pushButton:hover {
	background-color :rgb(35, 50, 60);
}</string>
            </property>
            <property name="text">
             <string>Acceuil</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/nodejs.png</normaloff>:/images/nodejs.png</iconset>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>45</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>7</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>150</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton_2 {
	background-color :rgb(26, 32, 41);
	border: none;
	color : white;
}
#pushButton_2:hover {
	background-color :rgb(35, 50, 60);
}</string>
            </property>
            <property name="text">
             <string>Etudiants invités à s'inscrire</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/nodejs.png</normaloff>:/images/nodejs.png</iconset>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>45</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>7</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>150</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton_3 {
	background-color :rgb(26, 32, 41);
	border: none;
	color : white;
}
#pushButton_3:hover {
	background-color :rgb(35, 50, 60);
}</string>
            </property>
            <property name="text">
             <string>Frais non honoré</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/nodejs.png</normaloff>:/images/nodejs.png</iconset>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>45</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>7</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>150</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton_4 {
	background-color :rgb(26, 32, 41);
	border: none;
	color : white;
}
#pushButton_4:hover {
	background-color :rgb(35, 50, 60);
}</string>
            </property>
            <property name="text">
             <string>Situation financière</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/nodejs.png</normaloff>:/images/nodejs.png</iconset>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>45</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>7</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>150</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton_5 {
	background-color :rgb(26, 32, 41);
	border: none;
	color : white;
}
#pushButton_5:hover {
	background-color :rgb(35, 50, 60);
}</string>
            </property>
            <property name="text">
             <string>Pédagogiquement autorisé</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/nodejs.png</normaloff>:/images/nodejs.png</iconset>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>45</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>7</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>150</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#pushButton_6 {
	background-color :rgb(26, 32, 41);
	border: none;
	color : white;
}
#pushButton_6:hover {
	background-color :rgb(35, 50, 60);
}</string>
            </property>
            <property name="text">
             <string>Demade d'échelonnement</string>
            </property>
            <property name="icon">
             <iconset resource="images.qrc">
              <normaloff>:/images/nodejs.png</normaloff>:/images/nodejs.png</iconset>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>200</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#frame { 
	border: none;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QFrame" name="frame_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#frame_2 {
	border: none;
	background-color: rgb(35, 50, 60);
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>10</y>
             <width>101</width>
             <height>40</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">#label_2 {
	color : white;
}</string>
           </property>
           <property name="text">
            <string>User name</string>
           </property>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#label {
	color : white;
	text-align : center;
	padding-left :  23px;
border: 1px solid black;
}</string>
          </property>
          <property name="text">
           <string>Logo</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page">
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>220</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>page 1</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_3">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTabWidget" name="levelTab">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="widget">
             <attribute name="title">
              <string>Tab 1</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_6">
              <item row="0" column="0">
               <widget class="QTabWidget" name="fieldTab">
                <widget class="QWidget" name="tab_6">
                 <attribute name="title">
                  <string>Tab 1</string>
                 </attribute>
                </widget>
                <widget class="QWidget" name="tab_7">
                 <attribute name="title">
                  <string>Tab 2</string>
                 </attribute>
                </widget>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_2">
             <attribute name="title">
              <string>Tab 2</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_3">
             <attribute name="title">
              <string>Page</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_4">
             <attribute name="title">
              <string>Page</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="tab_5">
             <attribute name="title">
              <string>Page</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_5"/>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_4">
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>300</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>page 3</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_5">
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>310</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>page 4</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_2">
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>250</y>
            <width>67</width>
            <height>17</height>
           </rect>
          </property>
          <property name="text">
           <string>page 5</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_6">
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>180</y>
            <width>92</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>CheckBox</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
